lms/resources/_js_config.py

```diff
@@ -26,8 +26,8 @@ def _consumer_key(self):
         return self._lti_user.oauth_consumer_key
 
     @property
-    def _ai_getter(self):
-        return self._request.find_service(name="ai_getter")
+    def _application_instance_service(self):
+        return self._request.find_service(name="application_instance")
```

Comment:
Might simplify things a little to make this return the application instance rather than the service, as we always call `get()` on it anyway.